packages/core/src/di/injectable.ts

```diff
@@ -64,9 +64,14 @@ export interface InjectableDecorator {
  */
 export interface Injectable {
   /**
-   * Determines which injectors will provide the injectable,
-   * by either associating it with an `@NgModule` or other `InjectorType`,
-   * or by specifying that this injectable should be provided in one of the following injectors:
+   * Determines which injectors will provide the injectable.
+   *
+   * - `Type<any>` - associates the injectable with an `@NgModule` or other `InjectorType`,
+   * - 'null' : Equivalent to `undefined`. The injectable is not provided in any scope automatically
+   * and must be added to a `providers` array.
```

Comment:
nit: I'd propose mentioning where the `providers` array can be found:
```suggestion
   * and must be added to a `providers` array of an [@NgModule](api/core/NgModule#providers), [@Component](api/core/Directive#providers) or [@Directive](api/core/Directive#providers).
```